### DISCUSSION TITLE
建议自定义加密模块

### DISCUSSION THREAD

**Discussion by DarkGiter** [SpenserCai/DRat#3] [Ideas]
建议可以自定义配置文件的加密模块

**Comment by SpenserCai** [comment]
事实上现在就可以，在DRat上你可以通过修改：config\manager.go中的DecryptEnsConfig函数来实现解密部分的修改该，同时在[DRatConfig](https://github.com/SpenserCai/DRatConfig)中修改对应的加密函数即可
